tests/issues/issue-289/unformatted.pug

```diff
@@ -0,0 +1 @@
+my-component( foo="bar",  #default="{  baz }" )
```

Comment:
@backbone87 Please provide an example that fails the test 🤔

Reply:
If you're using Vue 3, could you try to remove/disable `pugSingleFileComponentIndentation`? Just guessing that maybe there is a problem with that?! :thinking: But not sure.

Reply:
failing:

unformatted: `my-component(  #default="{ foo, bar: baz }")`
formatted: `my-component(#default="{ foo, bar: baz }")`

```
SyntaxError: Unexpected token ','

      301 |                             }
      302 |                     } catch (error: any) {
    > 303 |                             throw new Error(error);
          |                                   ^
      304 |                     }
      305 |                     token = this.getNextToken();
      306 |             }
```

Reply:
the test passes with the following change:
```ts
export function isVueExpression(name: string): boolean {
	return /^((v-(bind|slot))?:|v-(model|slot|if|for|else-if|text|html)|#).*/.test(name);
}
```
(added `|#`)